src/bundle/ui-dev/src/modules/universal-discovery/components/filters/filters.panel.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { getTranslator } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/context.helper';
+
+const FiltersPanel = ({ children, isApplyButtonEnabled, makeSearch, clearFilters }) => {
+    const Translator = getTranslator();
+    const filtersLabel = Translator.trans(/*@Desc("Filters")*/ 'filters.title', {}, 'ibexa_universal_discovery_widget');
+    const clearLabel = Translator.trans(/*@Desc("Clear")*/ 'filters.clear', {}, 'ibexa_universal_discovery_widget');
+    const applyLabel = Translator.trans(/*@Desc("Apply")*/ 'filters.apply', {}, 'ibexa_universal_discovery_widget');
+
+    return (
+        <div className="c-filters-panel">
+            <div className="c-filters-panel__header">
+                <div className="c-filters-panel__header-content">{filtersLabel}</div>
+                <div className="c-filters-panel__header-actions">
+                    <button className="btn ibexa-btn ibexa-btn--ghost ibexa-btn--small" type="button" onClick={clearFilters}>
+                        {clearLabel}
+                    </button>
+                    <button
+                        type="button"
+                        className="btn ibexa-btn ibexa-btn--secondary ibexa-btn--small ibexa-btn--apply"
+                        onClick={makeSearch}
+                        disabled={!isApplyButtonEnabled}
+                    >
+                        {applyLabel}
+                    </button>
+                </div>
+            </div>
+            {children}
+        </div>
+    );
+};
+
+FiltersPanel.propTypes = {
+    children: PropTypes.node.isRequired,
```

Comment:
total nitpick, not sure if props like this should be required... depends how you look on it - this component would work without it 100% after all. But at the same time, probably wouldn't make sense in context we're using it in.

Reply:
I assumed this component does not make sense without children, but on the other hand, I don't think such a requirement adds much value...